presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -292,13 +315,13 @@ public InputStreamSources createDictionaryStreamSources(Map<StreamId, Stream> st
             Map<StreamId, StreamCheckpoint> checkpoints = getStreamCheckpoints(includedOrcColumns, types, decompressor.isPresent(), rowGroupId, encodings, streams, columnIndexes);
             int rowOffset = rowGroupId * rowsInRowGroup;
             int rowsInGroup = Math.min(rowsInStripe - rowOffset, rowsInRowGroup);
-            rowGroupBuilder.add(createRowGroup(rowGroupId, rowOffset, rowsInGroup, valueStreams, checkpoints));
+            rowGroupBuilder.add(createRowGroup(rowGroupId, rowOffset, rowsInGroup, columnIndexes.entrySet().stream().mapToLong(e -> e.getValue().get(rowGroupId).getColumnStatistics().getMinAverageValueSizeInBytes()).sum(), valueStreams, checkpoints));
```

Comment:
Maybe extract a variable for this large chunk of new code and wrap it across multiple lines.